libs/datamodel/core/src/lib.rs

```diff
@@ -248,7 +248,11 @@ pub fn render_datamodel_to(
         .map(|c| c.preview_features())
         .unwrap_or_else(BitFlags::empty);
 
-    let lowered = LowerDmlToAst::new(datasource, preview_features).lower(datamodel);
+    let lowered = lower(LowerParams {
```

Comment:
why not `lower(datasource, datamodel)` and the preview features can be added if needed?

Reply:
I want a struct for the params because we're going to extend it. We could remove the preview features now, I just thought it's likely they come up again soon, but you're right it's a case for YAGNI, I'll remove the preview features.

Reply:
Explanation approved. BOSS DEFEATED